colordetect/colour_segmentation.py

```diff
@@ -0,0 +1,52 @@
+import cv2
+import numpy as np
+
+
+class ColourSegmentation:
+
+    @staticmethod
+    def get_image_array(image_path: str, resize_h: int = None):
```

Comment:
We could, instead of having another module still reading an image and parsing the image to the next method,
modify the `ColorDetect` `__init__` method to read the image? So in our case, the `get_segmented_image` would be part of the ColorDetect?  Does this seem feasible or might it bring maintainability problems in future?

Reply:
I'll check that

Reply:
```
import cv2
from colordetect import ColorDetect


color_detect = ColorDetect("assets/car.jpg", resize_h=200)
colour_count = color_detect.get_color_count()
color_detect.write_color_count()

output_image, gray, segmented, mask = color_detect.get_segmented_image(
    lower_bound=(0, 70, 0), upper_bound=(80, 255, 255), use_grab_cut=False
)

cv2.imshow("Image", color_detect.image_original)
cv2.imshow("Result", output_image)
cv2.imshow("Gray", gray)
cv2.imshow("Segmented", segmented)
cv2.imshow("Mask", mask)
if cv2.waitKey(0) == 27:
    cv2.destroyAllWindows()
```